CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - `--set-on` argument
 ### Changed
 - Default mode changed from `count-up` to `countdown`
+- `white-tea` program duration changed from `10 minutes` to `3 minutes`
```

Comment:
Isn't this format better:
`10` minutes to `3` minutes?

Reply:
Good point 👍 
Fixed in https://github.com/sepandhaghighi/mytimer/pull/110/commits/12609b936376c814e74e2c97d7712b79988c888f